content/bn/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,86 @@
+---
+reviewers:
+- lavalamp
+title: কুবারনেটিস কম্পোনেন্টস 
+content_type: concept
+description: >
+  কুবারনেটিস ক্লাস্টার তৈরী করে যে মূল কম্পোনেন্টস গুলো তাদের একটি ওভারভিউ। 
+weight: 10
+card:
+  title: Components of a cluster
+  name: concepts
+  weight: 20
+---
+
+<!-- overview -->
+
+এই পেইজটি প্রয়োজনীয় কম্পোনেন্টসগুলির একটি উচ্চ-স্তরের ওভারভিউ প্রদান করে যা একটি কুবারনেটিস ক্লাস্টার তৈরি করে।
+
+{{< figure src="/images/docs/components-of-kubernetes.svg" alt="Components of Kubernetes" caption="The components of a Kubernetes cluster" class="diagram-large" clicktozoom="true" >}}
```

Comment:
- You can localize the caption and `alt` attribute.
- You can localize the diagram itself.

Reply:
alt="একটি ক্লাস্টারের কম্পোনেন্ট গুলো"

Reply:
@sftim I can localize the diagram  . Which directory we should store the localized version of this svg image ?

Reply:
Localize in this[ PR](https://github.com/kubernetes/website/pull/48818)